configure.ac

```diff
@@ -11323,6 +11364,9 @@ echo "   * SIMD+FPU disable as flags:  $ASFLAGS_FPUSIMD_DISABLE" && \
 echo "   * SIMD+FPU enable as flags:   $ASFLAGS_FPUSIMD_ENABLE" && \
 echo "   * Linux kernel module PIE:    $ENABLED_LINUXKM_PIE"
 
+test "$ENABLED_BSDKM" = "yes" && \
+echo "   * FreeBSD Kernel Build Root:  $BSDKM_ROOT"
+
```

Comment:
Would want to add `Kernel mode: $KERNEL_MODE` and `Kernel mode defaults: $KERNEL_MODE_DEFAULTS`.